packages/components/src/range-control/stories/index.tsx

```diff
@@ -0,0 +1,248 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { starEmpty, starFilled, styles, wordpress } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import RangeControl from '..';
+
+const ICONS = { starEmpty, starFilled, styles, wordpress };
+
+const meta: ComponentMeta< typeof RangeControl > = {
+	component: RangeControl,
+	title: 'Components/RangeControl',
+	argTypes: {
+		afterIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		allowReset: { control: { type: 'boolean' } },
+		beforeIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		color: { control: { type: 'color' } },
+		disabled: { control: { type: 'boolean' } },
+		help: { control: { type: 'text' } },
+		initialPosition: { control: { type: 'number' } },
+		marks: { control: { type: 'object' } },
+		min: { control: { type: 'number' } },
+		max: { control: { type: 'number' } },
+		onChange: { action: 'onChange', control: { type: null } },
+		railColor: { control: { type: 'color' } },
+		showTooltip: { control: { type: 'boolean' } },
+		step: { control: { type: 'number' } },
+		trackColor: { control: { type: 'color' } },
+		withInputField: { control: { type: 'boolean' } },
+	},
+	parameters: {
+		controls: { expanded: true },
+		docs: { source: { state: 'open' } },
+	},
+};
+export default meta;
+
+const Template: ComponentStory< typeof RangeControl > = ( {
+	onChange,
+	...args
+} ) => {
+	const [ value, setValue ] = useState< number >();
+
+	return (
+		<RangeControl
+			{ ...args }
+			value={ value }
+			onChange={ ( v ) => {
+				setValue( v );
+				onChange?.( v );
+			} }
+		/>
+	);
+};
+
+export const Default: ComponentStory< typeof RangeControl > = Template.bind(
+	{}
+);
+Default.args = {
+	help: 'Please select how transparent you would like this.',
+	initialPosition: 50,
+	label: 'Opacity',
+	max: 100,
+	min: 0,
+};
+
+/**
+ * Setting the `step` prop to `"any"` will allow users to select non-integer
+ * values. This also overrides both `withInputField` and `showTooltip` props to
+ * `false`.
```

Comment:
I think we have a regression in the base branch of this PR, because this is now showing the tooltip!

<img width="250" alt="Story with any step is showing tooltip" src="https://user-images.githubusercontent.com/555336/171159350-3ff54dfb-addb-4ee9-bab6-45fd54fe6419.png">

Reply:
Good catch. Thank you! I've fixed this is the base branch.